goguen/secp256k1.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv
+, autoconf
+, automake
+, libtool
+, getSrc
+}:
+
+stdenv.mkDerivation rec {
+  name = "secp256k1";
+  src  = getSrc name;
+
+  buildInputs = [ autoconf automake libtool ];
+
+  configurePhase = ''
+    ./autogen.sh
```

Comment:
if you put `autoreconfHook` into the inputs, it will run autogen for you
then you can use normal `configureFlags` to set the params of `configure`

Reply:
I'm not even sure we'd need this at all.

Reply:
I'm looking towards removing the initial G-ala-C implementation at some point in the future.
We can't do it for a reason right now.
In this light, I'm not sure it'd be an economic expenditure of effort to focus on making this pretty.